### ISSUE TITLE
{Linked:Bug11605401}[Adaptive Card-Invoices] -No error message is provided for 'Invoices' and 'Billed Milestones' required field and form get submitted without filling this required field. 

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8833]
### Target Platforms

Android

### SDK Version

1.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Android Version: 14
App: Adaptive Cards Android Visualizer
App Version: 2023.12.21.1
Tool: Logitech Bluetooth Keyboard

**Repro Steps:**
1. Open Adaptive Cards Visualizer Application.
2. Navigate to import card using tab key and activate it using Enter key.
3. Navigate to the 'Choose file' combo box and press enter key to activate it.
4. Now add the json file.
5. Now Navigate to visual Tab and navigate to submit button and activate it.
6. Navigate the sample card and observe the issue.

**Actual Result:**
No error message is provided for "Invoice field is required" and "The milestone field is required" field and form get submitted without filling this required field. 

**Expected Result:**
The error should be identified, and error suggestion message should be provided for the "Invoice field is required" and "The milestone field is required" required field when it is not filled, also form should not be submitted as the required fields are empty in the respective form. 

User Experience:
Talkback users will not know there was an error until they encounter one of the indicators. They may abandon the form altogether before encountering the error indicator, thinking that the page simply is not functional. 

MAS Reference:
[MAS 3.3.1 – Error Identification](https://aka.ms/MAS3.3.1)

### Screenshots

https://github.com/microsoft/AdaptiveCards/assets/89897257/c186b3c9-cdf9-492b-82ff-9240af56d0ff

### Card JSON

```JSON
{
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "type": "AdaptiveCard",
    "version": "1.3",
    "body": [
        {
            "type": "ColumnSet",
            "columns": [
                {
                    "type": "Column",
                    "width": 2,
                    "items": [
                        {
                            "type": "TextBlock",
                            "text": "Please enter the below details to create cancellation and rebill request",
                            "weight": "Bolder",
                            "wrap": true
                        },
                        {
                            "type": "Input.ChoiceSet",
                            "id": "InvoiceNumber",
                            "style": "expanded",
                            "isMultiSelect": true,
                            "isRequired": true,
                            "errorMessage": "The invoice field is required.",
                            "label": "Invoices (Number - Amount(Currency))",
                            "choices": [
                                {
                                    "title": "Board - 100 USD",
                                    "value": "123"
                                },
                                {
                                    "title": "List - 2,000 USD",
                                    "value": "345"
                                },
                                {
                                    "title": "Assigned - 120 USD",
                                    "value": "234"
                                }
                            ],
                            "value": ""
                        },
                        {
                            "type": "Input.Number",
                            "label": "Total amount of expenses to be billed",
                            "wrap": true,
                            "isRequired": true,
                            "errorMessage": "The total amount of expenses to be billed is required.",
                            "id": "InvoiceAmount",
                            "value": 200
                        },
                        {
                            "type": "Input.ChoiceSet",
                            "id": "Milestones",
                            "style": "expanded",
                            "isMultiSelect": true,
                            "isRequired": true,
                            "errorMessage": "The milestone field is required.",
                            "label": "Billed Milestones (Description - Amount(Currency))",
                            "wrap": true,
                            "choices": [
                                {
                                    "title": "Milestone 1 - 100 USD",
                                    "value": "123"
                                },
                                {
                                    "title": "Milestone 2 - 2,000 USD",
                                    "value": "345"
                                },
                                {
                                    "title": "Milestone 3 - 120 USD",
                                    "value": "234"
                                }
                            ],
                            "value": ""
                        },
                        {
                            "type": "Input.Date",
                            "label": "Transaction from date",
                            "id": "FromDate",
                            "value": "2/12/2024"
                        },
                        {
                            "type": "Input.Date",
                            "label": "Transaction to date",
                            "id": "ToDate",
                            "value": "2/18/2024"
                        },
                        {
                            "type": "Input.ChoiceSet",
                            "id": "Reason",
                            "isRequired": true,
                            "errorMessage": "Please select a reason",
                            "label": "Reason",
                            "choices": [
                                {
                                    "title": "Reason 1",
                                    "value": "123"
                                },
                                {
                                    "title": "Reason 2",
                                    "value": "345"
                                },
                                {
                                    "title": "Reason 3",
                                    "value": "234"
                                }
                            ],
                            "value": ""
                        },
                        {
                            "type": "Input.Text",
                            "id": "Comments",
                            "label": "Comments",
                            "wrap": true,
                            "isMultiline": true,
                            "value": ""
                        }
                    ]
                }
            ]
        }
    ],
    "actions": [
        {
            "type": "Action.Submit",
            "title": "Submit",
            "data": {
                "id": "CancelAndRebillRequest_Submit"
            }
        }
    ]
}
```

**Comment by Raisul123** [comment]
Dependent Bug ID: 
https://dev.azure.com/microsoftit/_workitems/edit/11605401
